templates/express/files/.platform/services.yaml

```diff
@@ -0,0 +1,8 @@
+# The services of the project.
+#
+# Each service listed will be deployed
+# to power your Platform.sh project.
+
+mydatabase:
+    type: mysql:10.2
```

Comment:
Maybe use the `mariadb` here instead.

Reply:
That's how I had it before, but it resulted in the weirdness I mentioned earlier: I can use `mariadb` in type, but I would still need to use `mysql` for the endpoint.

Reply:
ahhh yeap, I remember that